src/NuGet.Core/NuGet.Packaging.Extraction/GlobalSuppressions.cs

```diff
@@ -7,7 +7,7 @@
 
 [assembly: SuppressMessage("Design", "CA1056:Uri properties should not be strings", Justification = "<Pending>", Scope = "member", Target = "~P:NuGet.Packaging.Core.RepositoryMetadata.Url")]
 [assembly: SuppressMessage("Design", "CA1012:Abstract types should not have constructors", Justification = "<Pending>", Scope = "type", Target = "~T:NuGet.Packaging.PackageReaderBase")]
-[assembly: SuppressMessage("Build", "CA2211:Non-constant fields should not be visible", Justification = "<Pending>", Scope = "member", Target = "~F:NuGet.Packaging.Licenses.NuGetLicenseData.LicenseListVersion")]
+[assembly: SuppressMessage("Build", "CA2211:Non-constant fields should not be visible", Justification = "Code is auto-generated by the GenerateLicenseList tool.", Scope = "member", Target = "~F:NuGet.Packaging.Licenses.NuGetLicenseData.LicenseListVersion")]
```

Comment:
The GenerateLicenseList tool is our own custom tool, and I believe lives in the same repo, so we could change it. However, it's part of the public API, which is a stronger reason why we possibly shouldn't change it. On the other hand, you introduced API breaking changes in NuGet.ProjectModel and NuGet.Versioning, so it seems like this field should be consistent.

Reply:
Fixed in the latest commit. updated `$\NuGet.Client\test\TestExtensions\GenerateLicenseList\LicenseDataCodeGenerator.cs` file so that this change won't be reverted in the future.